test/index.js

```diff
@@ -2486,4 +2486,16 @@ describe('Defaults', () => {
             done();
         });
     });
+
+    it('defaults inherits agents properties', (done) => {
+
+        const wreck1 = Wreck.defaults({});
+        const wreck2 = wreck1.defaults({});
+
+        //using === because `.to.equal(Wreck.agents)` triggers a deepEqual explosion
+        expect(wreck1.agents === Wreck.agents).to.equal(true);
```

Comment:
I didn't try that one but if this PR gets a 👍 otherwise I'd be glad to make the change